### PR TITLE
Common protobuf location

### PR DESCRIPTION
**Pull request by adamgering**
Moves protobufs to a common folder

### CODE REVIEW

File: rbac/processor/event_handler.py [hunk 1]
```diff
@@ -117,6 +120,7 @@ def namespaces(self):
         return [addresser.NS]
 
     def apply(self, transaction, state):
+        # try:
```
Comment (nadiabahrami) [inline rbac/processor/event_handler.py]: Is this needed?  If not, remove.

File: rbac/processor/event_handler.py [hunk 2]
```diff
@@ -152,6 +156,9 @@ def apply(self, transaction, state):
 
         else:
             raise InvalidTransaction("Message type unknown.")
+        # except Exception as exe:  # pylint: disable=broad-except
```
Comment (nadiabahrami) [inline rbac/processor/event_handler.py]: Is this needed?  If not, remove.

File: bin/protogen [hunk 3]
```diff
@@ -43,6 +43,4 @@ def make_protobuf(path, pkg_name):
 if __name__ == '__main__':
     make_protobuf('rbac/processor', 'protobuf')
     make_protobuf('rbac/transaction_creation', 'protobuf')
-    make_protobuf('rbac/server/api', 'protobuf')
-    make_protobuf('rbac/server/db', 'protobuf')
-    make_protobuf('rbac/ledger_sync', 'protobuf')
+    make_protobuf('rbac/common', 'protobuf')
```
Comment (nadiabahrami) [inline bin/protogen]: Why are the processor and transaction creation protobufs not moved to common?

**Comment by adamgering** [comment]
resolved @nadiabahrami comments, removed commented lines

**Pull request merged**